basics/LICENSE

```diff
@@ -0,0 +1,121 @@
+Creative Commons Legal Code
```

Comment:
Any reason you don't use the same license and the other code under the WebAssembly org (Apache)?

Reply:
I wanted to avoid any barrier to their use, whether it's potential GPLv2 incompatibility, the attribution requirement, or anything else. The main goal in the basics directory is to lay out interface details, rather than to write novel code. But I'm open to doing things differently.

I've set things up so that other directories can have other licenses (as the top-level LICENSE in the spec repo does), which we can use for third-party code, and also use if we do end up writing anything non-trivial.